docker/setup.sh

```diff
@@ -0,0 +1,4 @@
+#Cluster mode
+if [ "$mode" = cluster ]; then
+   mv /carapace/conf/cluster.properties /carapace/conf/server.properties;
```

Comment:
could we just make a copy of the file? 

having the "mode" as docker build parameter seems like you want to have 2 different docker images